test/jit/test_save_load.py

```diff
@@ -360,16 +360,11 @@ def _helper(m, fn):
                 else:
                     fn_result = self._try_fn(fn, a, b)
 
-                if not a.is_floating_point():
-                    # NOTE: Torchscript rewrites the module forward into
-                    #   torch.reciprocal(a) * b, but torch.reciprocal is
-                    #   implemented for integer dtypes.
-                    self.assertTrue(m_result, Exception)
-                    self.assertTrue('"reciprocal_cpu" not implemented for' in str(m_result))
-                elif isinstance(m_result, Exception):
-                    self.assertTrue(fn_result, Exception)
+                if isinstance(m_result, Exception):
+                    self.assertTrue(isinstance(fn_result, Exception))
                 else:
-                    self.assertEqual(m_result, fn_result)
+                    if fn is torch.div or a.is_floating_point():
```

Comment:
This is a good update (I like how the original wrote "torch.reciprocal is implemented" instead of "is not implemented"), but the cases need to be exhaustive. Currently it looks like they are only because an `if` is hiding in the `else` branch. It should be: 

```
if ...:
  ...
elif ...:
  ...
else:
  ...
```

Reply:
Changed to if...elif...else structure and added an explanation of why this case should be skipped in the latest revision